tests/pytests/test.py

```diff
@@ -3796,40 +3857,7 @@ def verify_address(addr):
                    'UNIXADDR',
                    '/tmp/redis.sock',
                    'MASTER'
-            ).error().contains('Invalid tcp address').contains(addr)
-
-def test_cluster_set_server_memory_tracking(env):
```

Comment:
why remove this test?

Reply:
It tests that the parser is not using the wrong allocator. Not relevant anymore